core/liquidity/v2/snapshot.go

```diff
@@ -450,25 +483,38 @@ func NewSnapshotEngine(
 	positionFactor num.Decimal,
 	slaParams *types.LiquiditySLAParams,
 ) *SnapshotEngine {
+	if slaParams == nil {
+		slaParams = defaultLiquiditySLAParams()
```

Comment:
@jeremyletang  I wonder if we still need this since you have added the defaults into to actual market snapshot.

Reply:
we'll keep as is for now.